addons/l10n_ro/__openerp__.py

```diff
@@ -22,16 +24,26 @@
 {
     "name" : "Romania - Accounting",
     "version" : "1.0",
-    "author" : "ERPsystems Solutions",
+    "author" : "ERPsystems Solutions, Fekete Mihai, Tatár Attila",
     "website": "http://www.erpsystems.ro",
     "category" : "Localization/Account Charts",
     "depends" : ['account','account_chart','base_vat'],
     "description": """
-This is the module to manage the Accounting Chart, VAT structure, Fiscal Position and Tax Mapping.
-It also adds the Registration Number for Romania in OpenERP.
-================================================================================================================
+Localization - Accounting base for Romania
+------------------------------------------
+
+Main features include:
+    - Accounting chart, taxes
+    - VAT structure 
+    - Registration Number from Chamber of Commerce and Industry
+    - Fiscal positions    
+
+    **~*~**
+Modulul conţine planul de conturi, cote TVA, poziţii fiscale. 
+Pentru a vă ajuta la instalare, sunt salvate screen-shot-uri cu setări în directorul 'inst_guide_images'.
+
+**Important**, pentru a avea funcţionalitatea contabilă completă, mai sunt necesare alte câteva module.
```

Comment:
What other modules? Do you mean the `account_accountant` module? This is true for any localisation, not really linked to Romanian.